recipe/meta.yaml

```diff
@@ -9,52 +9,50 @@ package:
 
 source:
   url: https://github.com/libffi/libffi/archive/v{{ version }}.tar.gz
-  sha256: 96d08dee6f262beea1a18ac9a3801f64018dc4521895e9198d029d6850febe23
-  patches:
-    - configure_includedir_option.patch
+  sha256: 3f2f86094f5cf4c36cfe850d2fe029d01f5c2c2296619407c8ba0d8207da9a6b
+  patches:  # [win]
+    #- configure_includedir_option.patch  # [win]
```

Comment:
Why comment out theses patches? It would be better to just remove them

Reply:
Done. removed them